doc/api/querystring.md

```diff
@@ -101,6 +101,10 @@ added: v0.1.25
 The `querystring.stringify()` method produces a URL query string from a
 given `obj` by iterating through the object's "own properties".
 
+This method serializes the following types of values passed in `obj`: strings,
```

Comment:
Maybe this paragragh can be added to the type annotation of `obj` above? Using the `{Type}` syntax would give us links to MDN documentations. Something like

```
Values should be {String|Number|Boolean|String[]|Number[]|Boolean[]}
(more explanations)
```